src/components/interactive-builder/modals/question/question.modal.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import QuestionModal from './question.modal';
+import { FormFieldProvider } from './form-field-context';
+import { showSnackbar } from '@openmrs/esm-framework';
+import type { FormField, RenderType } from '@openmrs/esm-form-engine-lib';
+import type { Schema } from '@types';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  showSnackbar: jest.fn(),
+  useDebounce: jest.fn((val) => val),
+}));
+
+jest.mock('@hooks/useConceptLookup', () => ({
+  useConceptLookup: jest.fn(() => ({ concepts: [], conceptLookupError: null, isLoadingConcepts: false })),
+}));
+
+jest.mock('@hooks/useConceptId', () => ({
+  useConceptId: jest.fn(() => ({ concept: null, isLoading: false, error: null })),
+}));
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key, defaultValue) => defaultValue || key,
+  }),
+}));
```

Comment:
Is this required? We already have a mock for `react-i18next` in https://github.com/openmrs/openmrs-esm-form-builder/blob/main/__mocks__/react-i18next.js